app/test_split_integer.py

```diff
@@ -2,20 +2,20 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    sum(split_integer(6, 2)) == 6
```

Comment:
You forgot to use the `assert` keyword to actually perform the test. The current line just calculates the sum and compares it, but it doesn't do anything with the result. It should be `assert sum(split_integer(6, 2)) == 6`.